### DISCUSSION TITLE
Why don't we convert our community calls to a podcast?

### DISCUSSION THREAD

**Discussion by claredillon** [InnerSourceCommons/InnerSourceMarketing#195] [Ideas]
A great question asked by @dellagustin  in Slack after our last community call. 

It's been asked/suggested below - and we have decided not to in the marketing group... here are some notes on the discussion included here for the record. 

Please do share/add more thoughts/insights if you have them - it's a decision that can be reviewed over time.

**Not insignificant cost/resources need to be committed.** 
We explored the idea in the past with some of the folks involved in other community podcasts. Successful podcasts typically have a regular cadence, with pods dropping at least on a monthly basis, often more often. In our experience - that kind of regular production requires structured program management (not just volunteer time) that would take additional resources at InnerSource Commons to ensure that happens on a regular basis. 

**Is the format right?**
The presentation bit we record for YouTube often refers to visuals which would not be visible on podcasts. The discussion bit is under Chatham House Rule, and can't be therefore shared. In my experience, the most successful podcasts formats are slightly different to presentations - more like our discussions. I love CHAOSScast and SustainOSS - both of whom have a panel chatting with a guest.

**Comment by claredillon** [comment]
Some great info from @dellagustin in Slack which will help us consider this moving forwards:

So, here is the idea - a good chunk of the value of the community calls can be taken from audio alone - people could for instance listen to that while they drive.
That can already be done with YouTube, my hypothesis here is that people are more used to do that from their podcast apps.
Making the community calls available as podcast also makes us present on an open ecosystem (which youtube is not).
Also, there is an open standard for podcast that can show images assigned to chapters. Still in adoption phase, but with many apps supporting it already (https://podcastindex.org/apps?appTypes=podcast+player&elements=Chapters).
The feed could be hosted directly in our page, as well as the required metadata (e.g. chapters are externalized as json files.
For audio hosting, we could use [archive.org](http://archive.org/), by my experience that is good enough.
If this is interest, I could do some experiments, kick it off and run it for some time, but on the long run I’d need someone to take over.
At least, I could create an issue for this in GH and describe the idea in more detail.

**Reply by claredillon** [reply]
Be really good to understand the effort required on an on-going basis. But there is a question about ROI on that effort vs. the idea of getting our community to go on other podcasts with an established audience.

**Reply by dellagustin** [reply]
Hi @claredillon , one comment about the question _Is the format right?_

The community calls are clearly not a primarily a podcast. Distributing them through RSS an making them available podcast players would be an alternative mean of content distribution. We can also experiment with distributing them in this way with video, not only audio, as many players support this.
Related to that, have you considered uploading the ISC videos to https://archive.org/ as an alternative mean of distribution?
I'm asking because I'm thinking about creating an RSS with some community calls so that we can have an idea about what is possible and how it would look like in the players.

**Comment by claredillon** [comment]
We do have connections with the folks who do both CHAOSScast and SustainOSS podcasts, and they have been great about sharing their experiences and processes.
